index.php

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>title</title>
```

Comment:
I would give your application a more appropriate title like `Collection` or `Netflix Series`

Reply:
Renamed to Netflix Series Collection